AdyenDropIn/Presentation/WrapperViewController.swift

```diff
@@ -49,8 +49,8 @@ internal final class WrapperViewController: UIViewController {
         child.view.translatesAutoresizingMaskIntoConstraints = false
         let topConstraint = child.view.topAnchor.constraint(equalTo: view.topAnchor, constant: 0)
         NSLayoutConstraint.activate([
```

Comment:
this could be `adyen.anchore`

Reply:
could be, and we get the top constraint by searching for it, but I don't like this approach :-).